pkg/controllers/controller.go

```diff
@@ -204,6 +202,11 @@ func InstallControllers(stopCh <-chan struct{}, aggregatorConfig *aggregatorapis
 			operatorInformerFactory.Operator().V1().Klusterlets(),
 		)
 
+		klog.Info("starting placementrule controller")
+		if err := placementrulecontroller.AddToManager(mgr); err != nil {
```

Comment:
put this to https://github.com/stolostron/multicluster-controlplane/blob/main/pkg/controllers/controller.go#L148 ?

Reply:
what is difference?